x/evm/types/utils.go

```diff
@@ -134,3 +135,42 @@ func BinSearch(lo, hi uint64, executable func(uint64) (bool, *MsgEthereumTxRespo
 func EffectiveGasPrice(baseFee, feeCap, tipCap *big.Int) *big.Int {
 	return math.BigMin(new(big.Int).Add(tipCap, baseFee), feeCap)
 }
+
+// scaleFactor constant for scaling 18 decimals <-> 6 decimals
```

Comment:
This could be moved  in` x/evm/wrappers/utils.go` after merging bank wrapper PR and use the `Decimals` method `ConversionFactor` to abstract the non-18 decimals value.